python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -39,23 +39,73 @@
 FOR_BODY_PREFIX = 'for_loop_body'
 GENERATE_VARIABLE_PREFIX = 'generate_variable'
 
+ATTRIBUTE_VARIABLE_PREFIX = '__attribute_variable'
 
-def create_while_node(condition_name, body_name, loop_var_names):
+
+def create_while_nodes(condition_name, body_name, loop_var_names):
+    """
+    Returns a list of gast.Node which represents the calling of Paddle
+    controlflow while_loop.
+
+    Usually, the list just contain 1 statement such as:
+
+    [a, b, c] = paddle.jit.dy2static.convert_while_loop(
+            condition_name, body_name, [a, b, c])
+
+    where a, b, c are in loop_var_names.
+
+    However, if loop_var_names contains attribute such as foo.x, we cannot
```

Comment:
Here I had a comment error: "attribute" -> "property"

I will change it in the near future (for example, after reviewer reviews it) because current commit passes CI tests.